test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -1442,7 +1443,7 @@ public void ScoreTransformerChainModel()
             Assert.NotNull(loadedData.Schema.GetColumnOrNull("Score"));
         }
 
-        //[Fact]
+        //[RetryFact]
```

Comment:
📝 Stale comment.

💡 Tests should be disabled with a `Skip` attribute, not by commenting out the code.